src/libs/ui/widgets/webviewtab.h

```diff
@@ -64,6 +64,9 @@ public slots:
     void activateSearchBar();
     void back();
     void forward();
+    void zoomIn();
```

Comment:
Please add an empty line above, since the slots are no longer sorted alphabetically.